CHANGELOG.md

```diff
@@ -10,7 +10,9 @@ _None_
 
 ### New Features
 
-_None_
+* All templates with bundle have now a new `spmBundle` parameter which allows usage of `Bundle.module` instead of `Bundle(for: BundleToken.self)` for SPM packages.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/c5c39037466e4ab1d35c70511ec11e6d3ec651e3/CHANGELOG.md#L13"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>